source/mainnet/net/mobile-wallet-gen2/setup-mobile-wallet.rst

```diff
@@ -87,9 +87,53 @@ Once the identity provider has verified your identity you can submit an account.
    .. image:: ../images/mobile-wallet-gen2/submit-acct.png
       :width: 25%
 
-.. Note::
-   To access the **Balance** of the new account, tap the Balance area on the account card or tap |morepage|.
+Basic navigation
+================
 
-.. |morepage| image:: ../images/more-ellipsis.png
-             :alt: Three dots button
-             :width: 40px
+In the wallet you have some navigation options.
+
+.. image:: ../images/mobile-wallet-gen2/main-nav.png
+      :alt: main wallet screen with position numbers
+      :width: 50%
+
+1. Balances: shows total balances of all accounts.
+2. Account: shows the identity on which the account was created, and totals.
+3. :ref:`Wallet settings<mw-gen2-wallet-settings>`: contains actions for the wallet.
+4. Scan QR code: tap to :ref:`scan a QR code to connect to a dApp<connect-app-bw>`.
+5. Add: to :ref:`add a new account<create-account>`.
+6. Account actions: range of actions you can perform on the account, such as send, receive, bake/delegate.
+
+.. _mw-gen2-wallet-settings:
+
+Wallet settings
+---------------
+
+In the Wallet settings |wallet-settings| you have the following options.
+
+- Your identity cards: view all identities in the wallet and details of the identities, :ref:`edit identity names<change-mw-id-name>`, and :ref:`create new identities<create-initial-account>`.
+- Address book: :ref:`manage your address book<address-book-mw>`.
+- Update passcode and biometrics: :ref:`change your wallet passcode or activate biometrics<change-passcode-mw>`.
+- Recovery: :ref:`recover your wallet<recover-wallet>`.
+
+Account navigation
+------------------
+
+When you tap on an account, you see the details for the specific account, including balances and transactions. The toolbar contains actions that can be performed on the account. Use the left and right arrows at the end of the toolbar to navigate the actions.
+
+.. image:: ../images/mobile-wallet-gen2/transaction-overview.png
+      :width: 50%
+
+- Send: :ref:`send funds<send-CCD-wallets>`
+- Receive: find and share your address
+- Earn: configure and manage :ref:`baking<baker-mw>` or :ref:`delegation<delegation>`
+- Scan QR code: scan QR code of a dApp, for example
+- Account settings: perform some less often used actions on an account
+   - Transfer filters: configure filters to show or hide rewards or finalization rewards
+   - Show shielded balance: Show the shielded balance tab on the account and the shielding action buttons
+   - Release schedule: :ref:`inspect a release schedule<inspect-release-schedule-mw>`
+   - Change account name: :ref:`customize account names<change-mw-acct-name>`
+   - Export private key: :ref:`export your private key<export-key>` for testing smart contracts, for example
```

Comment:
```suggestion
   - Export private key: :ref:`export your private key<export-key>` for testing smart contracts.
```
I don't think it can be used for anything else than scanning dApp QR codes